webui/react/src/components/ResponsiveTable.tsx

```diff
@@ -86,7 +86,7 @@ const ResponsiveTable: ResponsiveTable = ({
 
   return (
     <div ref={tableRef}>
-      <Spinner spinning={spinning}>
+      <Spinner conditionalRender spinning={spinning}>
```

Comment:
think it would be better to show a `SkeletonTable` type thing than to do `conditionalRender`